src/reporter/query_1T1ENA.py

```diff
@@ -24,6 +24,7 @@ def query_1T1ENA(entity_id,   # In Path
     See /entities/{entityId}/attrs/{attrName} in API Specification
     quantumleap.yml
     """
+    pdb.set_trace()
```

Comment:
@daminichopra slight oversight here? it looks like you removed `import pdb` in an earlier commit, so we should get rid of this too otherwise QL will die of a horrible death while running the tests---gosh you gotta love dynamic languages...